ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/contentfragment/v1/contentfragment/contentfragment.html

```diff
@@ -0,0 +1,22 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.fragment="com.adobe.cq.wcm.core.components.models.contentfragment.ContentFragment"
+     data-sly-use.template="core/wcm/components/commons/v1/templates.html"
+     data-sly-test="${fragment.name}" data-sly-use.cfTemplates="templates.html">
+    <sly data-sly-call="${cfTemplates.contentFragment @ fragment=fragment, fragmentPath=properties.fragmentPath, wcmmode=wcmmode}"></sly>
+</sly>
+<sly data-sly-test.hasModel="${properties.modelPath}" data-sly-set.emptyText="Configured" />
```

Comment:
Instead of only printing `Configured` we could write `Configured for <model>`

Reply:
Do you want to display the configured model info in <mode> ?

Reply:
No sure I think Model path or Model name (maybe including a link to the model).

Reply:
I was also thinking about this. The model path would be easy because it's saved in the property (usable in HTL) however that can be a bit long sometimes and surprising for the user because in the edit dialog we display the model title in the dropdown, not the path. Displaying the model title implies exposing it via a new Sling model while the current solution relies solely on the related Core WCM model interface proving only a custom implementation for it. 
The solution you see above avoids that and keeps it simple.
We can change that if needed.

Reply:
Ok it should maybe first extended in the WCM Core Components then to display the model title.

Reply:
The model title is displayed now. I've fixed also several other things, including your points.

Reply:
+1